tests/test_utils.py

```diff
@@ -85,19 +85,24 @@ def mock_check_index(mocker):
     return mocker.patch.object(utils, "check_index")
 
 
-def test_remote_index():
+def test_remote_index(httpserver):
```

Comment:
I think this was a part of another PR which is currently pending approval. so you should remove this file before merging to master

Reply:
I think that the other PR have to be merged first and then with this PR we won't have any problem :)

Reply:
I think @nkundiushuti might be right, in any case check if this is a conflict and if it is just remove this, and merge master to have the change integrated